src/plugins/azgenerator/TemplateAzureCliCommands.ts

```diff
@@ -37,13 +38,13 @@ export function GenerateAzureCliCommands(model: CodeModelAz): string[] {
                 output.push("        client_factory=" + cf_name + ")");
                 let groupinfos = model.CommandGroup_Name.split(' ');
                 let extraInfo = "";
-                if(groupinfos.length == 2) {
+                if(groupinfos.length == 2 && model.Extension_Mode == 'experimental') {
```

Comment:
Add similar logic when Extension_Mode is preview.

Reply:
I thought preview don't need to add this `is_preview=True` at the end of the command?

Reply:
If not added, it means GA.